src/database/event_schema.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+
+const eventSchema = mongoose.Schema({
+  startDate: Date,
+  endDate: Date,
+  title: String,
+  hall: String,
+  speaker: String,
+  imageUrl: String,
+  descriptionTitle: String,
+  DescriptionText: String,
+});
+
+const Events = mongoose.model('events', eventSchema);
+
+
+module.exports = Events;
```

Comment:
```node
export default mongoose.model('events', eventSchema);
```

You can worry about the name of your export wherever you're importing it, you don't have to restrict yourself to a certain name. Same goes fo `src/database/program_schema.js` & `src/database/users_schema.js`